Automattic/jetpack
# CLI: Add Changelogger Script call to Jetpack CLI

Adds changelogger script call to Jetpack CLI.

Closes #19120 

#### Changes proposed in this Pull Request:

* Adds changelogger script call to Jetpack CLI.

#### Jetpack product discussion



p9dueE-2A1-p2

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* check out this branch
* test out the CLI commands: 
	* `jetpack changelog add` should walk you through choosing a project before launching the changelogger.
	* `jetpack changelog add [project]` should start the prompts in the specific project file, e.g. `jetpack changelog add plugin`
	* `jetpack changelog add [project]/[name]` should immediately run the changelogger for the specified project, e.g. `jetpack changelog add plugin/jetpack`
	* Anything else should give you an error and exit.
* Make sure the changelog was added correctly.

Could probably do with some better error handling, but it relies on the returned status code from node.js for now.
